support/ebpf/types.h

```diff
@@ -555,7 +555,7 @@ typedef struct Trace {
   // Monotonic kernel time in nanosecond precision.
   u64 ktime;
   // The current COMM of the thread of this Trace.
-  char comm[COMM_LEN];
+  u8 comm[COMM_LEN];
```

Comment:
This and the above `CustomLabel` change are due to arm64 and amd64 default `char` differing on signedness and would cause the generated code check to fail on one or the other flavor due to inconsistent type.

It also greatly simplifies things to have this unconditionall `u8` which maps to `uint8` (and for which `byte` is an alias). These are directly usable as `[]byte`.